clients/search/data.go

```diff
@@ -0,0 +1,32 @@
+package search
+
+// Model represents a model returned by the search api
+type Model struct {
+	Count      int    `json:"count"`
+	Items      []Item `json:"items"`
+	Limit      int    `json:"limit"`
+	Offset     int    `json:"offset"`
+	TotalCount int    `json:"total_count"`
+}
+
+// Item represents a single hierarchy item returned by the search api
+type Item struct {
```

Comment:
I think the json tags for item should have omitempty in case no results are found, like so:

`Code               string  `json:"code,omitempty"`

The reason for not setting the model struct keys to omitempty is because there is value in those fields being set to 0, i.e. you can set a different message to the web user, no results found for search term